portafly/src/components/pages/accounts/AccountsTable.tsx

```diff
@@ -72,21 +57,11 @@ const AccountsTable: React.FunctionComponent = () => {
     setSortBy(index, direction, true)
```

Comment:
Sorting needs the correction no more:
```suggestion
    setSortBy(index, direction)
```

Reply:
Well, technically, that line is not part of this PR

Reply:
Precisely why I pointed it out, it must be changed because of https://github.com/3scale/porta/pull/1977/files#diff-89d3f403e9ab019e878ee0565aefb328L110.